var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -91,6 +91,7 @@ class Boost(Package):
     default_install_libs = set(['atomic',
                                 'chrono',
                                 'date_time',
+                                'exception',
```

Comment:
This works but it looks like the `exception` libs are supported back to boost `1.39` while Spack includes versions back to `1.34`. Could you add a line like

```
        if not spec.satisfies('@1.39.0:'):
            withLibs.remove('exception')
```

in the `install` method? There is a section with a number of similar statements.

Reply:
Sure, no problem.